<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black;
color: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRun">
     <property name="title">
      <string>Open...</string>
     </property>
     <addaction name="action_sql_file"/>
    </widget>
    <addaction name="actionSave_As"/>
    <addaction name="menuRun"/>
    <addaction name="actionExport"/>
    <addaction name="actionBackup"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuSQL_Functions">
     <property name="title">
      <string>Database</string>
     </property>
     <addaction name="actionINSERT"/>
     <addaction name="actionDELETE"/>
     <addaction name="actionUPDATE"/>
    </widget>
    <widget class="QMenu" name="menuFace_Images">
     <property name="title">
      <string>Images</string>
     </property>
     <addaction name="actionAdd"/>
     <addaction name="actionAddImg"/>
     <addaction name="actionDelete"/>
     <addaction name="actionDelete_User"/>
    </widget>
    <addaction name="menuSQL_Functions"/>
    <addaction name="menuFace_Images"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTable"/>
    <addaction name="actionImages"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfig_Files"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionEnrol_GUI"/>
    <addaction name="actionMonitor"/>
    <addaction name="actionSecurity"/>
    <addaction name="actionpgAdmin4"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_2"/>
   </widget>
   <widget class="QMenu" name="menuRun_2">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRecognition_Train"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuRun_2"/>
   <addaction name="menuSettings"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="action_sql_file">
   <property name="text">
    <string>SQL File</string>
   </property>
  </action>
  <action name="actionINSERT">
   <property name="text">
    <string>INSERT</string>
   </property>
  </action>
  <action name="actionDELETE">
   <property name="text">
    <string>DELETE</string>
   </property>
  </action>
  <action name="actionUPDATE">
   <property name="text">
    <string>UPDATE</string>
   </property>
  </action>
  <action name="actionEnrol_GUI">
   <property name="text">
    <string>Enrol</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="text">
    <string>Monitor</string>
   </property>
  </action>
  <action name="actionSecurity">
   <property name="text">
    <string>Security</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>View Help</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Take Photos</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete Images</string>
   </property>
  </action>
  <action name="actionAddImg">
   <property name="text">
    <string>Import Images</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImages">
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionpgAdmin4">
   <property name="text">
    <string>pgAdmin4</string>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete User</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionRecognition_Train">
   <property name="text">
    <string>Recognition Trainning</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionConfig_Files">
   <property name="text">
    <string>Config Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
